apps/web/components/Organization/OfficeCard/OfficeCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Box } from '@island.is/island-ui/core'
+import * as styles from './OfficeCard.treat'
+
+interface OfficeCardProps {
+  name: string
+  address: string
+  city: string
+  openingHours: string
+}
+
+export const OfficeCard: React.FC<OfficeCardProps> = ({
+  name,
+  address,
+  city,
+  openingHours,
+}) => {
+  return (
+    <>
+      <Box
+        borderRadius="large"
+        overflow="hidden"
+        background="blue100"
+        boxShadow="subtle"
+        border="standard"
+        borderColor="blue200"
+        marginBottom={[1, 1, 1]}
+        marginTop={[1, 1, 1]}
+        paddingX={[3, 3, 3]}
+        paddingY={[3, 3, 3]}
+      >
+        <h1 className={styles.title}>{name}</h1>
+        <p style={{ fontWeight: 300, marginTop: 14 }}>{address}</p>
+        <div style={{ paddingTop: 18, fontWeight: 'bold' }}>Opnunartími</div>
```

Comment:
Same regarding `Text` component
```suggestion
        <Text as='div' fontWeight='semiBold' paddingTop='p5'>Opnunartími</Text>
```
We should avoid using font weights that are not defined in our theme. in this case using "bold" which is equivalent to using 700 would result in the browser faux bolding the text because we do not have the defined font loaded for that weight